cloudify_cli/config/parser_config.py

```diff
@@ -1089,6 +1090,23 @@ def parser_config():
                 },
                 'handler': cfy.upgrade.upgrade
             },
+            'rollback': {
+                'help': 'rollback manager upgrade',
```

Comment:
Please make sure all help texts you add here are perfectly consistent with other help texts in the CLI.